rand_core/src/lib.rs

```diff
@@ -284,6 +284,28 @@ pub trait SeedableRng: Sized {
     /// partially overlapping periods.
     ///
     /// For cryptographic RNG's a seed of 256 bits is recommended, `[u8; 32]`.
+    ///
+    ///
+    /// # Implementing `SeedableRng` for RNGs with large seeds
+    ///
+    /// Note that the required traits `core::default::Default` and
+    /// `core::convert::AsMut<u8>` are not implemented for large arrays
+    /// `[u8; N]` with `N` > 32. To be able to implement the traits required by
+    /// `SeedableRng` for RNGs with such large seeds, the newtype pattern can be
+    /// used:
+    ///
+    /// ```ignore
+    /// pub struct MyRngSeed(pub [u8; N]);
+    ///
+    /// impl Default for MyRngSeed { ... }
+    /// impl AsMut<u8> for MyRngSeed { ... }
```

Comment:
With just a few more lines the example can be made complete. Maybe it is worth it to do so? (and then also does not need to be ignored)

Reply:
I expanded the example to actually compile. What do you think?

Reply:
Done.